packages/web-components/src/components/rux-monitoring-icon/rux-monitoring-icon.tsx

```diff
@@ -64,6 +65,7 @@ export class RuxMonitoringIcon {
                 id="rux-advanced-status__icon"
                 class="rux-advanced-status"
                 title={`${this.notifications} ${this.label} ${this.sublabel}`}
+                part="container"
```

Comment:
This shouldn't need one since this same can be accomplished by styling rux-monitoring-icon directly